html/strong/demos/view/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Как выглядит — <b> — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    body {
+      font-family: "Roboto", sans-serif;
+      font-size: 20px;
```

Comment:
```suggestion
      font-size: 24px;
```

Когда демо состоит из одной демки, можно ставить размер побольше :)